types/auth0/auth0-tests.ts

```diff
@@ -1596,3 +1596,6 @@ management.getDeviceCredentials({ user_id: 'user_id' }).then(deviceCredentials =
 management.getDeviceCredentials({ user_id: 'user_id' }, (err, deviceCredentials) => {
     deviceCredentials; // $ExpectType DeviceCredential[]
 });
+
+management.deleteDeviceCredential({ id: 'id' }).then(() => {});
+management.deleteDeviceCredential({ id: 'id' }, (err) => {});
```

Comment:
```suggestion
management.deleteDeviceCredential({ id: 'id' }, err => {});
```
please run default formatter for anything new added:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes (prettier)